identity/handler.go

```diff
@@ -118,7 +126,15 @@ type adminListIdentities struct {
 //       500: jsonError
 func (h *Handler) list(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 	page, itemsPerPage := x.ParsePagination(r)
-	is, err := h.r.IdentityPool().ListIdentities(r.Context(), page, itemsPerPage)
+
+	filters := map[string][]string{}
+	for k, v := range r.URL.Query() {
+		if strings.HasPrefix(k, "traits.") || strings.HasPrefix(k,"credentials.") {
+			filters[k] = v
```

Comment:
If you have multiple values for the same parameter, this will overwrite the most recently found parameter. Since `filters` is of type `map[string][]string`, this should be something like:
```
if _, ok := filters[k]; !ok {
  filters[k] = []string{}
}

filters[k] = append(filters[k], v)
```

Reply:
👍🏽